content/medo_manual/werken_met_deelnemers.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Werken met deelnemers
+sort: 300
+---
+
+* TOC
+{:toc}
+
```

Comment:
Ik zie nu pas dat dit in een alinea onderaan staat; ik zou het bovenaan zetten en nog toevoegen wat het verschil is tussen roqua en medo